src/pages/Host.tsx

```diff
@@ -4,20 +4,27 @@ import { GET_HOST } from "gql/queries/get-host";
 import { useQuery } from "@apollo/react-hooks";
 import { HostQuery, HostQueryVariables } from "gql/generated/types";
 import { usePageTitle } from "hooks/usePageTitle";
-import { PageWrapper } from "components/styles";
+import { PageWrapper, PageSider, PageLayout } from "components/styles";
 import { HostStatusBadge } from "components/HostStatusBadge";
 import { PageTitle } from "components/PageTitle";
 import { HostStatus } from "types/host";
+import { Metadata } from "pages/host/Metadata";
+import Code from "@leafygreen-ui/code";
 
 export const Host: React.FC = () => {
   const { id } = useParams<{ id: string }>();
   // Query host data
-  const { data, loading } = useQuery<HostQuery, HostQueryVariables>(GET_HOST, {
-    variables: { id },
-  });
+  const { data, loading, error } = useQuery<HostQuery, HostQueryVariables>(
+    GET_HOST,
+    {
+      variables: { id },
+    }
+  );
+
   const host = data?.host;
   const hostUrl = host?.hostUrl;
   const status = host?.status as HostStatus;
+  const sshCommand = "ssh mci@macos-1014-68.macstadium.build.10gen.cc";
```

Comment:
I think you accidently forgot this value here while testing you should use the real ssh command